src/main/php/xp/scriptlet/Develop.class.php

```diff
@@ -0,0 +1,71 @@
+<?php namespace xp\scriptlet;
+
+use lang\Runtime;
+use lang\RuntimeOptions;
+use util\cmd\Console;
+
+/**
+ * Serves requests using PHP's builtin webserver
+ *
+ * @see   http://php.net/manual/en/features.commandline.webserver.php
+ */
+class Develop extends \lang\Object {
+  private $host, $port, $url, $output;
+  
+  /**
+   * Creates a new instance
+   *
+   * @param  string $host
+   * @param  int $port
+   * @param  bool $verbose Whether to use verbose output. Optional, defaults to false.
+   */
+  public function __construct($host, $port, $verbose= false) {
+    $this->host= $host;
+    $this->port= $port;
+    $this->url= 'http://'.$host.':'.$port.'/';
+
+    if ($verbose) {
+      $this->output= function($line) {
+        Console::writeLine("  \e[36m", $line, "\e[0m");
+      };
+    } else {
+      $this->output= function($line) {
+        static $requests= 0;
+
+        Console::write("\r  \e[36m", $requests++, "\e[0m requests served @ ", date('r'));
+      };
+    }
+  }
+
+  /** @return string */
+  public function toString() {
+    return nameof($this).'(`'.Runtime::getInstance()->getExecutable()->getFileName().' -S`)';
+  }
+
+  /**
+   * Serve requests
+   */
+  public function serve(WebLayout $layout, $webroot, $profile, $docroot) {
+    $runtime= Runtime::getInstance();
+    $startup= $runtime->startupOptions();
+    $backing= typeof($startup)->getField('backing')->setAccessible(true)->get($startup);
```

Comment:
The `RuntimeOptions` class should have an accessor to inject further PHP command line arguments instead of us fiddling with the backing field directly.